pallets/idn-manager/src/lib.rs

```diff
@@ -279,20 +275,35 @@ pub type CreateSubParamsOf<T> = CreateSubParams<
 pub struct UpdateSubParams<SubId, Credits, Frequency, PulseFilter> {
 	// The Subscription Id
 	pub sub_id: SubId,
-	// New number of random values
+	// New number of pulses
```

Comment:
Since we have the option to charge for idle time, this comment is not always true right?